components/address/Address.module.css

```
@@ -145,50 +137,47 @@
+  color: black;
+  background-color: rgb(215, 215, 215);
+}
+.copyMenuItem:focus-visible {
+  outline: 2px solid #528bff;
+  outline-offset: -2px;
 }
 
 .addressTag {
-  font-family: variables.$main-font;
-  font-size: variables.$font-size;
+  font-family: "Montserrat", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)